### ISSUE TITLE
[TabStrip] "Failed to execute 'setAttribute'" error in logs

### ISSUE THREAD

**Issue by akonit** [telerik/kendo-angular#4251]
## Describe the bug
A strange error in logs when using kendo-tabstrip component:
```
Error: Failed to execute 'setAttribute' on 'Element': 'aria-disabled]' is not a valid attribute name.
```

## To Reproduce
Create any kendo-tabstrip-tab and check console logs.

https://stackblitz.com/edit/angular-gxdcsm - stackblitz example. Please check its console log to find this error

![kendo angular tabstrip error](https://github.com/telerik/kendo-angular/assets/6785911/09dd7122-0076-4dea-bf08-bda5ad387867)

## Expected behavior
This error should not be present in logs.

## Describe the bug

Package versions:
```
angular 17.3.0
kendo-angular 15.2.0
```

**Comment by slavenai** [comment]
Fixed in `v15.3.0` of `kendo-angular-layout` package.
